devtools/gradle/gradle-application-plugin/src/main/java/io/quarkus/gradle/Prompter.java

```diff
@@ -0,0 +1,85 @@
+package io.quarkus.gradle;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+import java.util.function.Consumer;
+
+import org.aesh.readline.Readline;
+import org.aesh.readline.ReadlineBuilder;
+import org.aesh.readline.tty.terminal.TerminalConnection;
+import org.aesh.terminal.tty.Signal;
+
+/**
+ * Prompt implementation.
+ *
+ * @author <a href="http://escoffier.me">Clement Escoffier</a>
```

Comment:
Same here, please get rid of the `@author` tags in a future version.

Reply:
ok

Reply:
On a follow up PR